File: MediaLibrary/MediaLibraryWebUI/lib/app/configurations/podcast-configuration.ts
﻿import BaseConfiguration from './base-configuration'
import * as Enums from '../assets/enums'

export default class PodcastConfiguration extends BaseConfiguration {
    private selectedPodcastId: number;
    private selectedPodcastPage: Enums.PodcastPages;
    private selectedPodcastSort: Enums.PodcastSort;
    private selectedPodcastFilter: Enums.PodcastFilter;

    constructor() {
        super();
    }
}

File: ML/ML_WebUI/Models/Configurations/PodcastConfiguration.cs
﻿using MediaLibraryWebUI.Models.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using static MediaLibraryWebUI.UIEnums;

namespace MediaLibraryWebUI.Models.Configurations
{
    public class PodcastConfiguration : BaseConfiguration
    {
        public PodcastConfiguration()
        {
        }

        public int SelectedPodcastId { get; set; }
        public PodcastPages SelectedPodcastPage { get; set; }
        public PodcastSort SelectedPodcastSort { get; set; }
        public PodcastFilter SelectedPodcastFilter { get; set; }
    }
}